src/reducers/breakpoints.js

```diff
@@ -182,8 +182,8 @@ export function getBreakpointsForSource(state: OuterState, sourceId: string) {
 export function getBreakpointForLine(
   state: OuterState,
   sourceId: string,
-  line: number
-) {
+  line: number | null
+): ?Breakpoint {
```

Comment:
The line is possibly `null`, and I had to explicitly type the result to make it optional, it inferred as `Breakpoint`.